data/data/bootstrap/baremetal/files/etc/containers/systemd/metal3-baremetal-operator.container

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=Metal3 deployment service
+BindsTo=ironic-volume.service
+Requires=build-metal3-env.service wait-iptables-init.service image-customization.service build-ironic-env.service
+Wants=network-online.target ironic-httpd.service ironic-ramdisk-logs.service
+After=network-online.target ironic-volume.service build-ironic-env.service image-customization.service build-metal3-env.service
+
+[Container]
+ContainerName=baremetal-operator
+Image=$METAL3_BAREMETAL_OPERATOR_IMAGE
+Exec=/baremetal-operator -build-preprov-image
+Network=host
+Volume=${AUTH_DIR}:/auth:z,ro
+Volume=/opt/openshift:/opt/openshift:z,ro
+Environment="XDG_RUNTIME_DIR=/run/user/${UID}"
+Environment="KUBECONFIG=/opt/openshift/auth/kubeconfig-loopback"
+
+# TODO: Change to file:/// after https://review.opendev.org/c/openstack/ironic/+/905626 merges
+Environment="DEPLOY_KERNEL_URL=http://127.0.0.1:6180/images/ironic-python-agent.kernel"
+
+Environment="PROVISIONING_INTERFACE=${PROVISIONING_INTERFACE}"
```

Comment:
nit: I don't remember if this is needed for BMO, possibly not

Reply:
Removed